!GameTexts/NewTexts.xml

```diff
@@ -1388,7 +1369,7 @@
 			<Text>Treasury of Global Treasures</Text>
 		</Row>
 		<Row Tag="TXT_KEY_WHITE_TOWER_THEMING_BONUS_HELP">
```

Comment:
Can you fix white Tower text bug "Hokath" mentioned?

Reply:
Treasury of Treasure, lol. could call it the "Royal {2_CivAdjective} Treasury"
"To maximize your bonus, bonus, make sure all slots are filled with Art from Different Eras and Civilizations."